code/modules/vehicles/_hitbox.dm

```diff
@@ -137,18 +137,15 @@
 	for(var/mob/living/tank_desant AS in tank_desants)
 		tank_desant.set_glide_size(root.glide_size)
 		tank_desant.forceMove(get_step(tank_desant, direction))
-		if(isxeno(tank_desant) || move_dist > 1)
```

Comment:
What's this?

Reply:
? it was a dupe check, its checked on the next line already.

Reply:
i am good at reading